app/clientsHandler.js

```diff
@@ -19,9 +20,11 @@ const CLIENTS_HANDLER = (() => {
      * This method registers a clientId to a specific game
      */
     const registerClient = (clientId, username, gameId) => {
+        const validToken = validateToken(token);
         REGISTERED_CLIENTS[clientId] = {
             gameId,
-            username
+            username,
+            loggedIn: validToken
```

Comment:
I understand that we need to track whether a client is logged in so we can determine whether we should try updating their stats. But I think that a guest user would have a valid token too, right? We should eventually change how this `loggedIn` variable is determined